wannierberri/__system.py

```diff
@@ -163,12 +179,24 @@ def getCC(self):
 
     @property
     def getSS(self):
-        return self.spin
+        return self.spin or (self.SHC and self.qiao)
 
     @property
     def getFF(self):
         return False
 
+    @property
+    def getSA(self):
```

Comment:
do you actually need getSA and getSHA incase ofiao method? I guessnot. Therefore I would suggest that you use two separate variables for the `System()` class initializer : SHCryoo and SHCqiao.